Sources/Logging/Logging.swift

```diff
@@ -60,11 +60,13 @@ extension Logger {
     public func log(level: Logger.Level,
                     _ message: @autoclosure () -> Logger.Message,
                     metadata: @autoclosure () -> Logger.Metadata? = nil,
+                    source: @autoclosure () -> String? = nil,
```

Comment:
@ktoso we _always_ resolve the autoclosures _before_ handing it off to the `LogHandler`. We also carefully place the `@inlinable`s so that the closure isn't actually created. `(() -> String))? = nil` does unfortunately not work because that's not an autoclosure anymore...

Reply:
Thanks, got it! Still sometimes unsure about these (my previous-life intuition still sometimes kicking in for those, there the `= nil` would have been better.)